moped-editor/src/components/GridTable/GridTableSearch.js

```diff
@@ -324,9 +325,7 @@ const GridTableSearch = ({
               {query.config.showExport && (
                 <Button
                   disabled={
-                    projectsList &&
-                    !projectsList["project_list_view"].length &&
-                    true
```

Comment:
It looks like we had a fixed table name under a dynamic component, it works fine under the projects list, but as this framework (GridTable and GridTableSearch) is used in other places like the Staff page, the keys would will not exist creating a WSOD.

Reply:
That makes a lot of sense!